src/templates/base.html

```diff
@@ -25,6 +25,10 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800">
     <link rel="stylesheet" href="/static/css/styles.css">
     {% endblock %}
+		{% set view_args = request.view_args.copy() %}
```

Comment:
The indentation here is inconsistent. Could you match the rest of the file?

Reply:
OK i'll correct.

Reply:
Looks like they're one nested level too many. Everything should be aligned with the blocks/links before it.

Reply:
Already corrected in the new commit no ?

Reply:
See screenshot below. New changes are too far indented:

![image](https://user-images.githubusercontent.com/1120896/60690546-1ff5db80-9e96-11e9-94ee-436eea6e06e0.png)

Reply:
Thanks. I think it ok now :)